src/app.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Form from './form.js'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      cards: []
+    }
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  handleSubmit(event) {
+    event.preventDefault()
+    const card = {}
+    card.question = document.querySelector('#questionInput').value
```

Comment:
You shouldn't need to query the `document` when building React apps. The `event.target` should contain everything you need.